### PR TITLE
Add Husky base to the platform without base planner

### PR DESCRIPTION
**Pull request by harryrobotics**
Update: passed the test with move_base planner.
Create the `husky_emmty.launch` in forlder `tests/husky`

I managed to add husky base without base planner to the platform.
It passed the make robot test and velocity control test on husky simulation. Note that the base planner is none for now.
TODO:
Need base planner topic name and to test position control 
Need Laser SLAM base to test SLAM

To reproduce the test please follow these steps:
1. 
```
sudo apt-get update
sudo apt-get install ros-kinetic-husky-desktop
sudo apt-get install ros-kinetic-husky-simulator
```
2.
`cd pyrobot/tests/husky`
`python run_husky_test.py`

### CODE REVIEW

**Comment by kalyanvasudev** [comment]
Thank you for adding Husky robot to PyRobot Harry. Your pull request will be approved soon after the review :)

File: src/pyrobot/core.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ def __init__(self,
         for f in os.listdir(cfg_path):
             if f.endswith('_config.py'):
                 robot_pool.append(f[:-len('_config.py')])
+        print(robot_pool)
```
Comment (viirya) [inline src/pyrobot/core.py]: I guess this is for debugging and should be removed?

Comment (harryrobotics) [reply src/pyrobot/core.py]: Thank you so much. I removed that.

**Comment by kalyanvasudev** [comment]
Hi Harry, could you please elaborate the purpose of this file? https://github.com/facebookresearch/pyrobot/blob/9b6cfdc2f65fd39b3a242800947160e056bd87a5/basics.html

**Comment by kalyanvasudev** [comment]
Also, it looks like you have a lot of repeated code from LoCoBot directory in https://github.com/facebookresearch/pyrobot/tree/9b6cfdc2f65fd39b3a242800947160e056bd87a5/src/pyrobot/husky

Could you please just change it to importing from LoCoBot directory in PyRobot? Here is an example on how you can do it - https://github.com/facebookresearch/pyrobot/blob/92d1d9c19148e3156869537c942a8820337d9d1a/tests/test_pyrobot_classes.py

**Comment by harry-savioke** [comment]
> Hi Harry, could you please elaborate the purpose of this file? https://github.com/facebookresearch/pyrobot/blob/9b6cfdc2f65fd39b3a242800947160e056bd87a5/basics.html

This file is generated when I run the test script which is modified based on your test script. I can remove it.

**Comment by harry-savioke** [comment]
> Also, it looks like you have a lot of repeated code from LoCoBot directory in https://github.com/facebookresearch/pyrobot/tree/9b6cfdc2f65fd39b3a242800947160e056bd87a5/src/pyrobot/husky
> 
> Could you please change it to importing from LoCoBot directory in PyRobot? Here is an example on how you can do it - https://github.com/facebookresearch/pyrobot/blob/92d1d9c19148e3156869537c942a8820337d9d1a/tests/test_pyrobot_classes.py

I don't want to import the controller from Locobot because I think each robot should have its own modified controller. I want to reduce the dependency between robots. So far, as I remembered, I don't need to change anything in the controller, but not sure in the future. How do you think?

**Comment by kalyanvasudev** [comment]
Closing this PR has the PyRobot API has changed since then. 
Please consider re-basing on the latest Develop branch.
If adding a new robot, please make sure that you have the necessary API has been tested on a real robot. 
Consider creating a tutorial similar to the Sawyer or LoCobot tutorial on our website. Thanks :)

**Pull request closed without merging**